CHANGELOG.md

```diff
@@ -4,13 +4,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ## StreamChatUI
+### ✅ Added
+- Unread messages separator can be enabled/disabled using `components.isUnreadMessagesSeparatorEnabled` (https://github.com/GetStream/stream-chat-swift/pull/2797)
```

Comment:
This is going to be a hotfix, so having an addition here is a bit weird 🤔 But I guess this one is a minor one and useful